README.md

```diff
@@ -1 +1,129 @@
+
+```
+$ cd Luxy-Rides-Frontend
+```
+
+Install the required dependencies to run the project with this command:
+```
+$ npm install
+```
+
+Then run it in your browser with this command:
+
+```
+$ npm start
+```
+
+Kindly modify the files as needed.
+
+### Run tests
+
+To run tests, please run this command
+```
+$ npm run test
+```
+## Authors
+
+👤 **Manel Hammouche** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [ha-manel](https://github.com/ha-manel)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [ha_manel_](https://twitter.com/ha_manel_)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [Manel Hammouche](https://www.linkedin.com/in/manel-hammouche/)
+
+👤 **Toussaint Saraza** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [sarazaToussaint](https://github.com/sarazaToussaint)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [toussaintSaraz1](https://twitter.com/ToussaintSaraz1)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [Toussaint Saraza](https://www.linkedin.com/in/toussaintsaraza/)
+
+👤 **Hammad Siddique** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [HammadSiddique](https://github.com/HammadSiddique)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [hs_devv](https://twitter.com/hs_devv)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [HammadSiddique](https://www.linkedin.com/in/-hammadsiddique/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- [Microverse](https://www.microverse.org/)
+- Original design idea by [Murat Korkmaz on Behance](https://www.behance.net/muratk).
+- [FontAwesome](https://fontawesome.com/) and [Flaticon](https://www.flaticon.com/) for icons.
+- [Unsplash](https://unsplash.com/) for images.
+
+## 📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Your project does not have a license file.  Please, kindly go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file, and add it to the root of your project.  After that, please include a link to your license file on your readme.

This is true for both the API and the frontend readmes.

That way, your project will be well documented 💪

Reply:
- [x] Done, thanks for pointing this out 👍